src/frontend/components/InstructorDashboardVOD/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Box } from 'grommet';
+import { normalizeColor } from 'grommet/utils';
+import React from 'react';
+import styled from 'styled-components';
+
+import { DashboardVideoLiveWidgetsContainer } from 'components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetsContainer';
+import { DashboardVideoLiveWidgetThumbnail } from 'components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetThumbnail';
+import { TeacherLiveInfoBar } from 'components/TeacherLiveInfoBar';
+import VideoPlayer from 'components/VideoPlayer';
+import { appData } from 'data/appData';
+import { DeleteTimedTextTrackUploadModalProvider } from 'data/stores/useDeleteTimedTextTrackUploadModal';
+import { InfoWidgetModalProvider } from 'data/stores/useInfoWidgetModal';
+import { useTimedTextTrack } from 'data/stores/useTimedTextTrack';
+import { useVideo } from 'data/stores/useVideo';
+import { initVideoWebsocket } from 'data/websocket';
+import { theme } from 'utils/theme/theme';
+import { TimedTrackModalWrapper } from './custom/TimedTrackModalWrapper';
+import { InstructorDashboardVODWidgetDownloadVideo } from './widgets/InstructorDashboardVODWidgetDownloadVideo';
+import { InstructorDashboardVODWidgetGeneralTitle } from './widgets/InstructorDashboardVODWidgetGeneralTitle';
+import { InstructorDashboardVODWidgetUploadClosedCaptions } from './widgets/InstructorDashboardVODWidgetUploadClosedCaptions';
+import { InstructorDashboardVODWidgetUploadSubtitles } from './widgets/InstructorDashboardVODWidgetUploadSubtitles';
+import { InstructorDashboardVODWidgetUploadTranscripts } from './widgets/InstructorDashboardVODWidgetUploadTranscripts';
+import { InstructorDashboardVODWidgetUploadVideo } from './widgets/InstructorDashboardVODWidgetUploadVideo';
+
+const StyledLiveVideoInformationBarWrapper = styled(Box)`
+  -webkit-box-shadow: 0px 0px 7px 5px ${normalizeColor('shadow-1', theme)};
+  box-shadow: 0px 0px 7px 5px ${normalizeColor('shadow-1', theme)};
+`;
+
+export const InstructorDashboardVOD = () => {
```

Comment:
Prefixing with `Instructor` is IMO useless. The dashboard is accessible only for instructors. Moreover, all other dashboard components are not prefixed with `Instructor`.